packages/core/src/components/toast/toaster.tsx

```diff
@@ -89,6 +93,9 @@ export class Toaster extends AbstractComponent<IToasterProps, IToasterState> imp
      * The `Toaster` will be rendered into a new element appended to the given container.
      */
     public static create(props?: IToasterProps, container = document.body): IToaster {
+        if (props != null && props.inline != null) {
```

Comment:
i'm just so used to doing it this way. 

how about if we tackle that change across the codebase when we enable `strictNullChecks`?